templates/emails/base_email.html

```diff
@@ -0,0 +1,162 @@
+       style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; background-color: #f6f6f6;">
+    <tr>
+        <td style="font-family: sans-serif; font-size: 14px; vertical-align: top;">&nbsp;</td>
+        <td class="container"
+            style="font-family: sans-serif; font-size: 14px; vertical-align: top; display: block; Margin: 0 auto; max-width: 580px; padding: 10px; width: 580px;">
+            <div class="content"
+                 style="box-sizing: border-box; display: block; Margin: 0 auto; max-width: 580px; padding: 10px;">
+
+                <!-- START CENTERED WHITE CONTAINER -->
+                <table class="main"
+                       style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%; background: #ffffff; border-radius: 3px;">
+
+                    <!-- START MAIN CONTENT AREA -->
+                    <tr>
+                        <td class="wrapper"
+                            style="font-family: sans-serif; font-size: 14px; vertical-align: top; box-sizing: border-box; padding: 20px;">
+                            <table border="0" cellpadding="0" cellspacing="0"
+                                   style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;">
+                                <tr>
+                                    <td style="font-family: sans-serif; font-size: 14px; vertical-align: top;">
+                                        {% block content %}{% endblock %}
+                                    </td>
+                                </tr>
+                            </table>
+                        </td>
+                    </tr>
+
+                    <!-- END MAIN CONTENT AREA -->
+                </table>
+
+                <!-- START FOOTER -->
+                <div class="footer" style="clear: both; Margin-top: 10px; text-align: center; width: 100%;">
+                    <table border="0" cellpadding="0" cellspacing="0"
+                           style="border-collapse: separate; mso-table-lspace: 0pt; mso-table-rspace: 0pt; width: 100%;">
+                        <tr>
+                            <td class="content-block"
+                                style="font-family: sans-serif; vertical-align: top; padding-bottom: 10px; padding-top: 10px; font-size: 12px; color: #999999; text-align: center;">
+                                <span class="apple-link" style="color: #999999; font-size: 12px; text-align: center;">
+                                    Testprojekt
+                                </span>
+                                <br> Keine Lust mehr auf Emails?
+                                <a href="http://i.imgur.com/CScmqnj.gif"
+                                   style="text-decoration: underline; color: #999999; font-size: 12px; text-align: center;">
+                                    dumm gelaufen, dieser Link funktioniert nicht
```

Comment:
See above. I will replace the placeholder text with Lorem Ipsum to make this clearer. I simply didn't consider unsubscription/email blacklisting within the scope of this issue. Let me know if I should add it.

Reply:
for logging, the ELK stack feels like industry standard. I have never implemented it, though, nor do I have any solid preferences. Whichever way we go, orchestrating the logging via Kubernetes seems ideal, but I haven't used Kubernetes yet, either.

It would probably be good to have a broad idea in terms of a  long-term setup including continuous integration, orchestration, configuration management, backup strategies etc., but for now I'd say it's really just prototyping to see if the service is actually viable. If Papertrail/Airbrake works, perhaps we stick to that for now, use pingdom for uptime and just deploy the docker container by hand?